comms/question_test.go

```diff
@@ -0,0 +1,33 @@
+package comms
+
+import (
+	"io/ioutil"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestQuestion(t *testing.T) {
+	tests := []struct {
+		desc     string
+		given    string
+		fallback string
+		expected string
+	}{
+		{"records response", "hello\n", "", "hello"},
```

Comment:
Wondering if the value for desc should be a little more descriptive and added as the message to assert.Equal to give a little context in case a test case fails, due to some future change. Thoughts?